### PR TITLE
fix nginx version in system requirement

### PR DESCRIPTION
**Pull request by lalittmohan**
## Purpose of this pull request

This pull request (PR) fix Nginx version on system requirement page

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements.html
- https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements.html

## Links to Magento source code

- N/A

### CODE REVIEW

File: src/_data/codebase/v2_4/system-requirements.yml [hunk 1]
```diff
@@ -10,13 +10,15 @@
   Redis: '6.2'
   Varnish: '7.0'
   Apache: '2.4'
-  nginx: '1.8'
+
```
Comment (BarnyShergold) [inline src/_data/codebase/v2_4/system-requirements.yml:13]: Remove all empty lines that were added to this file

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Review by lalittmohan — COMMENTED** [review]
fix space

**Review by BarnyShergold — APPROVED** [review]
All good! Thanks

**Review by jeff-matthews — COMMENTED** [review]
Thanks @lalittmohan. I just need to confirm with internal staff before merging.

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@dobooth the branch with the code is successfully imported into the `magento-devdocs/devdocs` repository. Branch name: `imported-lalittmohan-devdocs-9456`.

**Pull request merged**